app/components/flash_message_component.rb

```diff
@@ -1,14 +1,18 @@
-# frozen_string_literal: true
 class FlashMessageComponent < ViewComponent::Base
   ALLOWED_PRIMARY_KEYS = %i[info success warning].freeze
+  DEVISE_PRIMARY_KEYS = { alert: :warning, notice: :info }.freeze
 
   def initialize(flash:)
     super
     @flash = flash.to_hash.symbolize_keys!
   end
 
   def message_key
-    flash.keys.detect { |key| ALLOWED_PRIMARY_KEYS.include?(key) }
+    key =
```

Comment:
Does the CSS for the component still render correctly with Devise flash keys? The `classes` method generates a CSS class based on this key.